src/Jobs/UploadConvertedAnnotationsJob.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace Biigle\Modules\Ptp\Jobs;
+use Biigle\ImageAnnotation;
+use Biigle\ImageAnnotationLabel;
+use Biigle\User;
+use Biigle\Jobs\Job as BaseJob;
+use Exception;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Bus\Batchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Storage;
+/**
+ * Upload the result of  computing the expected area for Point to Polygon Conversion to the DB
+ */
+class UploadConvertedAnnotationsJob extends BaseJob implements ShouldQueue
+{
+    use Batchable, InteractsWithQueue, Queueable, SerializesModels;
+    /**
+     * The queue to push this job to.
+     *
+     * @var string
+     */
+    public $queue;
+
+    /**
+     * User responsible for the Point to Polygon conversion.
+     *
+     * @var User
+     */
+    public User $user;
+
+    /**
+     * The queue to push this job to.
+     *
+     * @var string
+     */
+    public string $inputDir;
+
+
+    public function __construct(string $inputDir, User $user)
+    {
+        $this->inputDir = $inputDir;
+        $this->user = $user;
+    }
```

Comment:
With new PHP syntax you can do:

```suggestion
    /**
     * @var $user User responsible for the Point to Polygon conversion.
     * @var $inputDir ...
     */
    public function __construct(public string $inputDir, public User $user)
    {
       //
    }
```